crates/rooch-rpc-api/src/jsonrpc_types/move_types.rs

```diff
@@ -384,6 +384,26 @@ impl From<H256View> for H256 {
     }
 }
 
+#[serde_as]
+#[derive(Default, Debug, Clone, PartialEq, Eq, Hash, Serialize, Deserialize, JsonSchema)]
+pub struct H256ViewResult(
```

Comment:
H256View is a String. I think we should replace it with StrView<H256>. We do not need to distinguish H256 argument and H256Result.